### ISSUE TITLE
<nav> element should have aria-label

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wai-tutorials#298]
http://www.w3.org/WAI/tutorials/page-structure/sections/

"It can be used multiple times on a page."
CODE SNIPPET:
`<nav role="navigation">
  …
</nav>`

consider this instead, 
"It can be used multiple times on a page, but if used more than once, each should have a label to distinguish it from other navigation sections on the page."
CODE SNIPPET:
`<nav role="navigation" aria-label="secondary">
  …
</nav>`
